src/gmt_cpt_masters.h

```diff
@@ -69,23 +69,23 @@
 "rainbow     : Rainbow: magenta-blue-cyan-green-yellow-red [C=HSV]",
 "red2green   : Polar scale from red to green via white [S, C=RGB]",
 "relief      : Wessel/Martinez colors for topography [R=-8000/+8000, H, C=RGB]",
-"roma        : Perceptually uniform \'seis\' colormap, by Fabio Crameri [C=RGB]",
+"roma        : Perceptually uniform \'seis\' colormap, by Fabio Crameri [S, C=RGB]",
 "romaO       : Perceptually uniform cyclic colormap by Fabio Crameri [C=RGB]",
 "seafloor    : Purple-blue-white bathymetry scale [R=-6000/0, C=RGB]",
 "sealand     : Smith bathymetry/topography scale [R=-6000/+3000, H, C=HSV]",
 "seis        : R-O-Y-G-B seismic tomography colors [C=RGB]",
 "split       : Like polar, but via black instead of white [S, C=RGB]",
 "srtm        : Like dem2, but with blue for oceans [H, C=RGB]",
 "terra       : Colors for global bathymetry/topography relief [R=-7000/7000, H, C=RGB]",
-"tofino      : Perceptually uniform bimodal colormap, dark, by Fabio Crameri [S,C=RGB]",
+"tofino      : Perceptually uniform bimodal colormap, dark, by Fabio Crameri [S, C=RGB]",
 "tokyo       : Perceptually uniform colormap without black or white, by Fabio Crameri [C=RGB]",
 "tokyoS      : Perceptually uniform categorical colormap without black or white, by Fabio Crameri [C=RGB]",
 "topo        : Sandwell/Anderson colors for topography [R=-7000/+7000, H, C=HSV]",
 "turku       : Perceptually uniform colormap by Fabio Crameri [C=RGB]",
 "turkuS      : Perceptually uniform categorical colormap by Fabio Crameri [C=RGB]",
 "turbo       : Google's Improved Rainbow Colormap for Visualization [C=RGB]",
-"vik         : Perceptually uniform bimodal colormap, light, by Fabio Crameri [S,C=RGB]",
-"vikO        : Perceptually uniform bimodal cyclic colormap, light, by Fabio Crameri [S,C=RGB]",
+"vik         : Perceptually uniform bimodal colormap, light, by Fabio Crameri [S, C=RGB]",
+"vikO        : Perceptually uniform bimodal cyclic colormap, light, by Fabio Crameri [S, C=RGB]",
```

Comment:
```suggestion
"vikO        : Perceptually uniform bimodal cyclic colormap, light, by Fabio Crameri [C=RGB]",
```
Are soft hinges possible with cyclic colormaps :thinking:?